File: frontend/src/types/Funcionario.ts
export interface Funcionario {
  funcionariosId: number
  nome: string | null
  codigoAntigo: string | null
  dataCadastro: string
  dataUltimoRegistro: string
}

export interface FuncionarioInput {
  nome: string
  codigoAntigo?: string | null
}

File: backend/FarmaciaEliseu.Api/Models/Funcionario.cs
namespace FarmaciaEliseu.Api.Models;

public class Funcionario
{
    public int FuncionariosId { get; set; }
    public string? Nome { get; set; }
    public string? CodigoAntigo { get; set; }
    public DateTime DataCadastro { get; set; }
    public DateTime DataUltimoRegistro { get; set; }
    public bool Deletado { get; set; }
}
